README.md

```diff
@@ -63,10 +63,11 @@ file, run the following command in your terminal:
 npx hardhat faucet
 ```
 
-To install a standalone faucet, run the following command in your terminal:
```

Comment:
should we describe any constraints on the faucet - verification, limits, etc.? or is that just in the CLI output

Reply:
a5ee023